ting_covers_backside.strongarm.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @file
+ * ting_covers_backside.strongarm.inc
+ */
+
+/**
+ * Implements hook_strongarm().
+ */
+function ting_covers_backside_strongarm() {
+  $export = array();
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'ting_backside_covers_modes';
+  $strongarm->value = array(
+    'ting_cover_small' => 'ting_cover_small',
+    'ding_primary_large' => 'ding_primary_large',
```

Comment:
What does these image profiles represent?
This module should only utilize _ting_cover_small_, _ting_cover_medium_ and _ting_cover_large_ image profiles.